pyproject.toml

```diff
@@ -0,0 +1,60 @@
+[tool.poetry]
+name = "ixctl"
+repository = "https://github.com/fullctl/ixctl"
+version = "0.1.0"
+description = "Full control of your environment"
```

Comment:
"ix control"

Reply:
Fixed in https://github.com/fullctl/ixctl/pull/38/commits/659f65d46701275c329e6588c6f4351224771d77